packages/cypress/src/add-commands/real-hover.ts

```diff
@@ -0,0 +1,275 @@
+/**
+ * NOTE: the `realHover` command is originally being implemented in `cypress-real-events` package.
+ * https://github.com/dmtrKovalenko/cypress-real-events/blob/develop/src/commands/realHover.ts
+ *
+ * However, due to known issues with conflicting types between Cypress and Jest, importing the `cypress-real-events`
+ * package here will cause such issues with TypeScript as our `@commercetools-frontend/cypress` package
+ * is checked and built together with all other packages and not in isolation.
+ * See https://docs.cypress.io/guides/tooling/typescript-support#Clashing-types-with-Jest.
+ *
+ * Therefore, we are porting here the implementation of `realHover` to avoid importing it from the
+ * original package `cypress-real-events`.
+ */
```

Comment:
As explained here, we can't import the `cypress-real-events` package directly due to conflicting Cypress vs Jest types.

We basically have 2 options:
1. Port the implementation of the commands we need (e.g. `realHover`) as I did here.
2. Assume that the `realHover` command is "available" but it must be defined / imported into the Cypress setup by the consumers (so not part of the `@commercetools-frontend/cypress` package).

Option 1 will require us to maintain the implementation. Option 2 requires less maintenance but the consumer must import the package `cypress-real-events`.

@CarlosCortizasCT @tdeekens  Any preferences?

Reply:
I don't like us maintaining more code 😅 but I prefer that over giving more work to consumers in this case.
So I'd would keep the implementation locally.

_(Hopefully we won't need to touch it often)_

Reply:
I would like to co-locate things where it's lifecycle occurs. In this case the menu and the ability to test it and how to should be co-located and come from App Kit. I would assume that this code only changes if Cypress changes its APIs (e.g. breaking change) which they have historically not done often. If we do not add it here we have to add it to many repositories and maintain it there while keep cross-repo consumption and test-setup in sync has been costly I think having the "mess" here is our best option.